### ISSUE TITLE
Type tool options missings

### ISSUE THREAD

**Issue by openanim** [opentoonz/opentoonz#4554]
### Is your feature request related to a problem? Please describe

It would be nice if we could bring back these options from the type tool. Their shortcuts seem to be available but the options are not available. Maybe they were never there in opentoonz.

<img width="874" alt="type tool missing options" src="https://user-images.githubusercontent.com/29790512/193061718-70296cfb-2560-47e6-a167-793c043daa81.png">

### Describe the solution you'd like

make available missings options

**Comment by RodneyBaker** [comment]
I've wondered about this in the past but haven't takent the time to investigate.
What I find interesting is that some of the fonts have those extra options while others do not.
This suggests to me that it is possible that the other font styles are not present or active on the system.

To test this theory try a really common font such as Times New Roman where the extra font types should appear:

![image](https://user-images.githubusercontent.com/2576432/193079287-985373b6-66a3-417b-a8bd-6640fd225fb6.png)

If these options do appear for some fonts then the next step would be to pick a font that doesn't have those options showing and then double check to make sure those are actively installed on the system.  

In other words, each of those font styles are separate font files and each must be activated/installed on the system separately.  Once installed Opentoonz should automatically recognize them.

**Comment by openanim** [comment]
@RodneyBaker Thank you a lot, it seems that you have seen right, by adding the missing fonts of the same family, OT can see them all (over 10). What I find strange is that at times the detection is different between some software for the same font. But it's another thing. 

So i can close it as it's already there !
